examples/examples_nodejs_test.go

```diff
@@ -674,52 +642,75 @@ func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
 
 // TestAccCNIAcrossUpdates tests that the CNI manifest is reapplied when the EKS provider changes its base manifest.
 func TestAccCNIAcrossUpdates(t *testing.T) {
-	t.Log("Running `pulumi up` with v2.1.0 of the EKS provider")
```

Comment:
This test was broken before. It didn't actually test the upgrade, but just deployed two different clusters...